contrib/opencensus-ext-azure/README.rst

```diff
@@ -179,6 +203,63 @@ Below is a list of standard metrics that are currently available:
 - Process CPU Usage (percentage)
 - Process Private Bytes (bytes)
 
+You can pass a callback function to the exporter to process telemetry before it is exported. Your callback function can return `False` if you do not want this envelope exported. Your callback function must
```

Comment:
You want this text duplicated?

Reply:
It's fine having the text duplicated, we have duplicate documentation for creating an Azure Monitor resource and how to instrument with the `instrumentation_key` for each section. Usually users come to this page looking for a specific section (either trace, metrics or logs) so this shouldn't be a eyesore.